datafusion/core/src/lib.rs

```diff
@@ -382,13 +382,13 @@
 //!
 //! Calling [`execute`] produces 1 or more partitions of data,
 //! as a [`SendableRecordBatchStream`], which implements a pull based execution
-//! API. Calling `.next().await` will incrementally compute and return the next
+//! API. Calling [`.next()`]`.await` will incrementally compute and return the next
```

Comment:
```suggestion
//! API. Calling [`next()`]`.await` will incrementally compute and return the next
```
![CleanShot 2024-11-15 at 10 55 52@2x](https://github.com/user-attachments/assets/6cfee250-9892-4164-aeb4-f98394cdbd92)